### PR TITLE
simplify the editor cells

### PR DESCRIPTION
**Pull request by hamilton**
the move to iframes drastically simplifies the requirements for an editor cell. For now, we merely need a container > row > editor.

Will assign a reviewer and change the title when this is ready to go.

### CODE REVIEW

File: src/components/cells/markdown-cell.jsx [hunk 1]
```diff
@@ -52,14 +44,6 @@ export class MarkdownCellUnconnected extends React.Component {
             }}
```
Comment (bcolloran) [inline src/components/cells/markdown-cell.jsx]: not going to block merge, but i think containerStyle is not needed anymore either

Comment (hamilton) [reply src/components/cells/markdown-cell.jsx]: yep already on it. note the WIP :)

Comment (bcolloran) [reply src/components/cells/markdown-cell.jsx]: oh sorry thought this was ready to merge! looks good btw -- i was actually about to merge it...

Comment (bcolloran) [reply src/components/cells/markdown-cell.jsx]: ping me when you're ready

Comment (hamilton) [reply src/components/cells/markdown-cell.jsx]: Almost there! Got side-tracked by the engagement survey

**Comment by hamilton** [comment]
@bcolloran feel free to merge at your convenience, barring anything else stopping the show.

**Pull request merged**